lib/syncgateway.py

```diff
@@ -23,3 +23,6 @@ def start(self, config):
 
     def restart(self, config):
         orchestration.syncgatewayactions.restart(self.host_name, config)
+
+    def __str__(self):
```

Comment:
If this implemented for debugging purposes then I would recommend using __repr__